modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilder.java

```diff
@@ -221,31 +221,33 @@ private List<String> _collectSassFiles(String dirName, File baseDir)
 	}
 
 	private long _getNewestModifiedTime(String baseDir, String[] fileNames) {
-		return Stream.of(
-			fileNames
-		).map(
-			fileName -> Paths.get(baseDir, fileName)
-		).map(
-			FileUtil::getLastModifiedTime
-		).max(
-			Comparator.naturalOrder()
-		).orElse(
-			Long.MIN_VALUE
-		);
+		List<Long> lastModifiedTimes = TransformUtil.transform(
```

Comment:
@slnn You missed to merge these `TransformUtil` usages.

Reply:
Done, resend in https://github.com/liferay-core-infra/liferay-portal/pull/3665